### PR TITLE
Feature/39942 - Catalog (w/o solr)

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39942

### CODE REVIEW

File: Magento_Review/templates/helper/summary_short.phtml [hunk 1]
```diff
@@ -7,35 +7,19 @@
...
-    </div>
+    <?php if ($rating):?>
+        <div
+            class="rating__star"
+            title="rating <?= /* @escapeNotVerified */ $rating ?>%"
+            style="width:<?= /* @escapeNotVerified */ $rating ?>%"
+        >
+            <span class="rating__indicator"></span>
+        </div>
+    <?php endif;?>
```
Comment (szafran89) [inline Magento_Review/templates/helper/summary_short.phtml]: missing space

File: Magento_Review/templates/helper/summary_short.phtml [hunk 2]
```diff
@@ -7,35 +7,19 @@
...
 <?php elseif ($block->getDisplayIfEmpty()): ?>
     <div class="product-reviews-summary short empty">
         <a class="action add"
-           href="<?= /* @escapeNotVerified */  $urlForm; ?>"
+           href="<?= /* @escapeNotVerified */  $urlForm ?>"
```
Comment (szafran89) [inline Magento_Review/templates/helper/summary_short.phtml]: leave in one line if less then 80 or move attrs t new line

File: Magento_Review/templates/helper/summary_short.phtml (hunk 1 above)
Comment (Aekal) [reply Magento_Review/templates/helper/summary_short.phtml]: Added

File: Magento_Review/templates/helper/summary_short.phtml (hunk 2 above)
Comment (Aekal) [reply Magento_Review/templates/helper/summary_short.phtml]: Done

**Pull request merged**